resources/output_strings.js

```diff
@@ -0,0 +1,536 @@
+// TODO: make this a static class with static members
+//       this requires babel-eslint or being patient.
+
+// Throw exceptions on invalid property names to catch easy typos.
+class ThrowOnInvalidProxy {
+  constructor(obj) {
+    this.obj = obj;
+
+    if ('toJSON' in obj)
+      throw new Error('Cannot have toJSON property.');
+
+    return new Proxy(this, {
+      set(target, property, value) {
+        throw new Error('Cannot set readonly object.');
+      },
+
+      get(target, name) {
+        if (name === 'toJSON')
+          return JSON.stringify(obj);
+
+        if (name in target.obj)
+          return target.obj[name];
+
+        throw new Error(`Unknown property ${name}`);
+      },
+    });
+  }
+}
+
+// General guidelines:
+// * property names should closely match English text
+// * use OnPlayer suffix for things with `${player}`
+// * use OnTarget suffix for things with `${name}`
+// * any other parameters (of which there are none, currently) should use consistent suffixes.
+// * the value of each property should be a single object with localized keys
+const OutputStrings = new ThrowOnInvalidProxy({
```

Comment:
Also not sure that `OutputStrings` is the best name here, but I think it kind of fits with the usage?

Reply:
I like `Outputs`, following the pattern of `Conditions`, `Responses`, `Outputs`, but I don't feel strongly.